### ISSUE TITLE
Local field pages slow to load

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#5077]
Some local field pages, like [17.17.20.2](https://beta.lmfdb.org/padicField/17.17.20.2), take over 10 seconds to load.  The underlying problem seems to be the following database query:
```
SELECT "id" FROM "nf_fields" WHERE "local_algs" @> ARRAY['2.12.24.194']::text[] LIMIT 1 ran in  12.184074878692627s
```
There is a GIN index, `nf_fields_local_algs_gin`, but it's not getting used:
```
EXPLAIN ANALYZE SELECT "id" FROM "nf_fields" WHERE "local_algs" @> ARRAY['2.12.24.194']::text[] LIMIT 1;
                                                        QUERY PLAN                                                        
--------------------------------------------------------------------------------------------------------------------------
 Limit  (cost=0.00..269.13 rows=1 width=8) (actual time=11598.245..11598.249 rows=1 loops=1)
   ->  Seq Scan on nf_fields  (cost=0.00..1650327.00 rows=6132 width=8) (actual time=11598.242..11598.243 rows=1 loops=1)
         Filter: (local_algs @> '{2.12.24.194}'::text[])
         Rows Removed by Filter: 18636015
 Planning Time: 5.421 ms
 Execution Time: 11598.304 ms
```

**Comment by AndrewVSutherland** [comment]
The page https://www.lmfdb.org/padicField/3.10.5.2 is also ridiculously slow.

**Comment by edgarcosta** [comment]
Resolved via #5116
